doc/en/developer/source/quickstart/intellij.rst

```diff
@@ -72,4 +72,11 @@ Access GeoServer front page
 ---------------------------
 
 * After a few seconds, GeoServer should be accessible at: `<http://localhost:8080/geoserver>`_
-* The default ``admin`` password is ``geoserver``.
\ No newline at end of file
+* The default ``admin`` password is ``geoserver``.
+
+Run GeoServer from Intellij on Windows
+---------------------------
```

Comment:
Make sure this underline is the same length as the line above, otherwise the docbuild will throw a warning or fail.

Reply:
I'm a little confused by this comment.  The underline looks exactly the same as the preceding one.  None the less, I have copied the preceding underline.

Reply:
Not the same as the previous underline, but the same as the text that you are trying to underline.

Reply:
For more information, see http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections